src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -1026,6 +1026,73 @@ public virtual Task<bool> TryUpdateModelAsync<TModel>([NotNull] TModel model)
                 predicate);
         }
 
+        /// <summary>
+        /// Validates the specified <paramref name="model"/> instance using values from the controller's current
+        /// <see cref="IValueProvider"/>.
+        /// </summary>
+        /// <param name="model">The model instance to update.</param>
+        /// <returns>A <see cref="bool"/> that on completion returns <c>true</c> if the ModelState is valid</returns>
+        [NonAction]
+        public virtual bool TryValidateModel(object model)
+        {
+            return TryValidateModel(model, null /* prefix */);
```

Comment:
user prefix: null
/* prefix */ is the syntax from pre C# 4.0 where named parameters where not available